packages/map-template/src/components/LocationDetails/LocationDetails.scss

```diff
@@ -6,7 +6,7 @@
     display: grid;
     height: 100%;
     overflow-y: hidden;
-    grid-template-rows: 40px 1fr 56px;
+    grid-template-rows: 40px 1fr 64px;
```

Comment:
It seems the 56px value is also used on line 16. Should that be changed as well?

Reply:
✅